stdlib/2.7/__builtin__.pyi

```diff
@@ -534,8 +534,8 @@ class dict(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     def has_key(self, k: _KT) -> bool: ...
     def clear(self) -> None: ...
     def copy(self) -> Dict[_KT, _VT]: ...
-    def get(self, k: _KT, default: _VT = None) -> _VT: ...
-    def pop(self, k: _KT, default: _VT = ...) -> _VT: ...
+    def get(self, k: _KT, default: _T = None) -> Union[_VT, _T]: ...
+    def pop(self, k: _KT, default: _T = ...) -> Union[_VT, _T]: ...
```

Comment:
I think pop() will need to be overloaded -- with one arg there's no possibility of it returning the default value (unlike get()).